pages.es/linux/yay.md

```diff
@@ -0,0 +1,37 @@
+# yay
+
+> Yet Another Yogurt: crea e instala paquetes desde el repositorio de usuarios de Arch.
+> Vea también `pacman`.
+> Más información: <https://github.com/Jguer/yay>.
+
+- Busca e instala paquetes de forma interactiva desde los repositorios y AUR:
+
+`yay {{nombre_del_paquete|término_de_búsqueda}}`
+
+- Sincroniza y actualiza todos los paquetes desde los repositorios y AUR:
+
+`yay`
+
+- Sincroniza y actualiza solo paquetes del AUR:
+
+`yay -Sua`
+
+- Instala un nuevo paquete desde los repositorios y AUR:
+
+`yay -S {{paquete}}`
+
+- Elimina un paquete instalado y tanto sus dependencias como archivos de configuración:
```

Comment:
```suggestion
- Elimina un paquete instalado en tanto sus dependencias como archivos de configuración:
```

Reply:
The word-by-word translation on this one is kinda tricky. Not sure if the whole sentence makes sense with this correction.

What about:
` - Elimina un paquete instalado, sus dependencias y archivos de configuración` ?